README.md

```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications  
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme. 
```

Comment:
- The / should also be a comma
- change "opportunity" to "opportunities"
- comma before "which"
- comma after "etc."